DataManagementSystem/Client/FTS3Operation.py

```diff
@@ -479,7 +482,7 @@ def _callback(self):
         # TODO: are we really ever going to change type... ?
         opFile.ChecksumType = 'ADLER32'
         opFile.Size = ftsFile.size
-        res = returnSingleResult(targetSE.getURL(ftsFile.lfn, protocol='srm'))
+        res = returnSingleResult(targetSE.getURL(ftsFile.lfn, protocol=registrationProtocols))
```

Comment:
Just double checking: this `registrationProtocols` is a list, what was before was a string. Still OK? What will be used?

Reply:
yes it still OK. It will iterate over the elements of the list to find the first working protocol. In practice it has very little importance, since this url is just what is put in the catalog, and then never used..